connector/profiler/src/main/scala/org/apache/spark/executor/profiler/package.scala

```diff
@@ -31,7 +31,7 @@ package object profiler {
 
   private[profiler] val EXECUTOR_PROFILING_DFS_DIR =
     ConfigBuilder("spark.executor.profiling.dfsDir")
-      .doc("HDFS compatible file-system path to where the profiler will write output jfr files.")
+      .doc("HDFS compatible file-system patxh to where the profiler will write output jfr files.")
```

Comment:
When you test like this, you had better use GitHub's `Draft` feature. :)

Reply:
it's a typo, I jsut wrongly pressed the keyboard ...